From 979be592b52b9e063b49f43550d2a259188dbbd7 Mon Sep 17 00:00:00 2001
From: Daniel Micay <danielmicay@gmail.com>
Date: Mon, 2 Apr 2018 12:55:28 -0400
Subject: [PATCH 045/230] tethering: use Cloudflare DNS servers as defaults

Cloudflare DNS has a better privacy policy than Google Public DNS while
still supporting DNS over TLS.
---
 .../android/networkstack/tethering/TetheringConfiguration.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java b/packages/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
index 90eb638ec31..42533d846b9 100644
--- a/packages/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
+++ b/packages/Tethering/src/com/android/networkstack/tethering/TetheringConfiguration.java
@@ -68,7 +68,7 @@ public class TetheringConfiguration {
         "192.168.48.2", "192.168.48.254", "192.168.49.2", "192.168.49.254",
     };
 
-    private static final String[] DEFAULT_IPV4_DNS = {"8.8.4.4", "8.8.8.8"};
+    private static final String[] DEFAULT_IPV4_DNS = {"1.0.0.1", "1.1.1.1"};
 
     /**
      * Override enabling BPF offload configuration for tethering.
-- 
2.25.1

